docs/using-the-badge/replacing-parts.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Replace badge components
+weight: 5
+---
+
+If you need to replace your badge components, you can order a replacement part from ebay, aliexpress etc.
+
+## Replace your screen
+
+The badge uses a GC9A01 1.28" round LCD module.
+
+Some links to the correct screen:
+
+- [Aliexpress with connector soldered on](https://www.aliexpress.com/item/1005004392903184.html)
+- [Aliexpress with loose connector](https://www.aliexpress.com/item/1005005936868343.html)
+- [Amazon](https://www.amazon.co.uk/ARCELI-Arduino-Resolution-Interface-Display/dp/B0CFXVD9HX/)
+
+Carefully pull the connector of the display up at a 90 degree angle. If you bought a new display with a connector that is soldered on, carefully push the connector of the new display into the slots. The display is held in place by friction, so this is all you need to do.
+
+If you bought a display with a loose connector you have to also push the display onto the connector. If you have soldering tools, you can permanently affix the display to the connector.
+
+## Replace the FPC connector
+
+The badge uses a [JUSHUO AFC01-S30FCC-00](https://www.lcsc.com/product-detail/FFC-FPC-Connectors_JUSHUO-AFC01-S30FCC-00_C262279.html).
+
+
+!!! tip "You've found a badge CHALLENGE!"
+
+    Your challenge, should you choose to accept it, is to document how you replaced your FPC if you do so (pictures would be great!). To see more information and accept the challenge (that is, comment on the issue), see this [issue](https://github.com/emfcamp/badge-2024-documentation/issues/175).
+
+## Replace the display cable
```

Comment:
Would put this section ahead of the ones that require precision soldering, as cable replacement is much easier. Also, the battery is in the way of replacing the cable, and detaching the battery from the front board is hard work and undocumented for the moment. (requires removing the display, gently heating up the PCB and then twisting, not bending, the battery off the glue pads).